MapboxGeocoder/MBGeocodeOptions.swift

```diff
@@ -80,6 +80,11 @@ open class GeocodeOptions: NSObject {
      */
     internal var queries: [String] = []
     
+    /**
+     The query mode of the forward or reverse geocoding request.
+    */
+    internal var mode: String = "mapbox.places"
```

Comment:
Nit: `String` can be inferred from the value on the right-hand side, so it can be removed.